pkg/gofr/gofr.go

```diff
@@ -296,6 +297,9 @@ func (a *App) Migrate(migrationsMap map[int64]migration.Migrate) {
 
 func (a *App) initTracer() {
 	traceExporter := a.Config.Get("TRACE_EXPORTER")
+	tracerURL := a.Config.Get("TRACER_URL")
+
+	// deprecated : tracer_host and tracer_port is deprecated and will be removed in upcoming versions
 	tracerHost := a.Config.Get("TRACER_HOST")
```

Comment:
Maybe you could report an error message when non-empty

Reply:
currently, this should not be an error as same could be achieved by providing `TRACER_HOST` and `TRACER_PORT`, but yeah since we are deprecating those.. will add a warn log instead.